### PR TITLE
Make clear that Windows is not supported for multi-node grid

### PR DESCRIPTION
**Pull request by manics**
Windows on a multinode system most likely doesn't won't work, and isn't going to be fixed:
https://trello.com/c/uR1VOttF/623-omero-node-should-be-refactored

### CODE REVIEW

File: omero/sysadmins/grid.txt [hunk 1]
```diff
@@ -24,6 +24,9 @@ Requirements
 ^^^^^^^^^^^^
 
 The normal OMERO installation actually makes use of OMERO.grid internally.
+Note that multi-node OMERO.grid configurations are only tested with Unix-like
+systems, multi-node configurations on Windows are untested and deprecated,
+and any remaining support will be dropped in OMERO 5.3.
 If you have followed the instructions under :doc:`unix/server-installation`
 (Unix) or :doc:`windows/server-installation` (Windows), then you
```
Comment (hflynn) [inline omero/sysadmins/grid.txt]: Can you remove this reference to the windows installation guide too please?

Comment (manics) [inline omero/sysadmins/grid.txt]: I originally removed that, then put it back in after I realised this page is a bit ambiguous, saying OMERO.grid can be used on the same host as well as across multiple hosts (and there's still an `OMERO.grid on Windows` section on this page). @joshmoore ?

Comment (joshmoore) [inline omero/sysadmins/grid.txt]: I'm find having the link to windows removed as @hflynn suggests, but if we need to be more radical in this paragraph (and potentially eliminate the "on Windows" section) we can

Comment (hflynn) [inline omero/sysadmins/grid.txt]: How about remove this link for now and then once we have a 5.3 branch, removing all reference to Windows on this page in a new PR on that branch?

Comment (sbesson) [inline omero/sysadmins/grid.txt]: @hflynn: feel free to use the `regions` branch as the place to open 5.3 PRs

Comment (hflynn) [inline omero/sysadmins/grid.txt]: Ah, there we go, the branch already exists!

**Comment by manics** [comment]
@hflynn Removed

**Comment by hflynn** [comment]
:+1:

**Comment by hflynn** [comment]
I'll be removing all the Windows sysadmin content for 5.3 anyway, I'll just add cleaning up the rest of this page to my list.

**Pull request merged**